index.js

```diff
@@ -0,0 +1,79 @@
+// create a `validatePassword` function which will return true when 
+// the password given meets all the listed requirements listed below.
+
+const validatePassword = (valPass) => {
+    
+        }
+
+const length = (valPass) => {
+    if (length.valPass > 8){
+        return true
+ }
+     else return false
+
+     }
+ 
+
+const lower = (valPass) => {
+    let lower = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q','r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
+    for (let i = 0; i< valPass.length; i++){
+        if (lower.indexOf(valPass[i]) >=0){
+            return true
+        }
+        else return false
+       
+    }
+
+}
+
+const UPPER = (valPass) => {
+    let UPPER = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K','L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z']
+    for (let i = 0; i< valPass.length; i++){
```

Comment:
you have UPPER and Upper so none of that worked.  ALWAYS use proper camel case to avoid this, we talked about this.  you should always do first word lower case and next ones following upper case that is what camel case is.  That ALSO broke your code